manifests/certonly.pp

```diff
@@ -94,6 +94,22 @@
     $default_args = '--text --agree-tos --non-interactive delete'
   }
 
+  # Plugin requirements
```

Comment:
I don't like duplicating the plugin selector. I'd prefer an `if $plugin == 'nginx' { require letsencrypt::plugin::nginx }` inside the default block.